schema/src/main/scala/io/shiftleft/codepropertygraph/schema/Method.scala

```diff
@@ -86,17 +85,6 @@ object Method extends SchemaBase {
       .mandatory(false)
       .protoId(221)
 
-    val parameterIndex = builder
-      .addProperty(
-        name = "PARAMETER_INDEX",
-        valueType = ValueType.Int,
-        comment = """
-            |Specifies the parameter's number. For the `this` parameter, the parameter index `0` is used.
-            |""".stripMargin
-      )
-      .mandatory(PropertyDefaults.Int)
-      .protoId(222)
-
```

Comment:
Removing this would mean all frontends need to be updated in one go or are we currently just ignoring unknown properties while loading?

Reply:
I'm removing this property, which I introduced yesterday and which isn't used yet.

Reply:
Correct. Can be removed.